### DISCUSSION TITLE
Deploy extensions settings as well?

### DISCUSSION THREAD

**Discussion by rieje** [yokoffing/Betterfox#278] [Q&A]
I run multiple (sandboxed) profiles for better privacy and organized tabs. Previously I was using a shell script that copied essential files like `extension-preferences.json extension-settings.json handlers.json extensions.json addonStartup.json.lz4 prefs.js` from a template profile to all other profiles; however, these files contain profile-specific identifiers like an extension's unique internal UUID. There's also some weird quirks when the template profile removes/adds extensions and that change gets pushed to the profiles (by simply copying over these files).

Sideloading extensions look like it was deprecated but I found about policies.json from this repository which lets you add extensions to auto-install on a new profile. 

Is it possible to deploy extensions setings as well? I would think that this is possible because it would be weird for Firefox to only allow extensions but not their extensions to be deployed. Firefox files only get created when it's launched (i.e. `firefox -CreateProfile` do not create these files), so my intended workaround to copy over extensions settings to `.mozilla/firefox/main/storage/default` and/or `.mozilla/firefox/main/browser-extension-data` (?) after launching the new profile and quitting it which will prompt the new files to be generated along with the install of extensions.

Any tips/suggestions are much appreciated.

**Comment by yokoffing** [comment]
I'm not sure. I know that Librewolf makes some changes to uBlock Origin's default filters. You may look at their repo and see how they implement that.

**Reply by iam-py-test** [reply]
Maybe relevant: https://github.com/gorhill/uBlock/wiki/Deploying-uBlock-Origin
Thanks

**Reply by yokoffing** [reply]
Maybe: https://www.reddit.com/r/uBlockOrigin/comments/o7q2ou/comment/h3cplhd/
